### PR TITLE
Delayed trigger

### PR DESCRIPTION
**Pull request by MBlesken**
DRAFT PR: Adjusting countdown clock description for SE

### CODE REVIEW

File: docs/2_2_common_mechanisms.adoc [hunk 1]
```diff
@@ -661,8 +661,7 @@ The importer calls <<fmi3GetClock>> to inquire the clock activation state.
 |`countdown`
 |[[countdown,`countdown`]]<<fmi3SetClock>> in <<EventMode>>,
 <<fmi3ActivateModelPartition>> in <<ClockActivationMode>>,
-<<fmi3GetInterval>> in <<InitializationMode>> and in every <<EventMode>>,
-<<fmi3GetInterval>> in <<ClockActivationMode>> after every call to <<fmi3ActivateModelPartition>>
+<<fmi3GetInterval>> in <<InitializationMode>>, in every <<EventMode>>, and in <<IntermediateUpdateMode>>
```
Comment (IZacharias) [inline docs/2_2_common_mechanisms.adoc:664]: Can you add here : In Scheduled Execution the FMU has to inform the importer by calling fmi3CallbackIntermediateUpdate that the clock is about to tick.

File: docs/5_1_scheduled_execution_math.adoc [hunk 2]
```diff
@@ -1,15 +1,13 @@
 === Concepts [[concepts-scheduled-execution]]
 
 The Scheduled Execution interface has a different timing concept compared to FMI for Co-Simulation.
-This is required to cover <<clock>> ticks for <<aperiodic-clocks,aperiodic>> <<inputClock,`input clocks`>> which may tick at time a instant that is unpredictable in advance for the simulation algorithm.
+This is required to cover ticks for <<clocks>> of <<causality, `causality == input`>> which may tick at time a instant that is unpredictable in advance for the simulation algorithm.
```
Comment (IZacharias) [inline docs/5_1_scheduled_execution_math.adoc:4]: I think "Triggered input clocks" are meant here.

Comment (andreas-junghanns) [reply docs/5_1_scheduled_execution_math.adoc:4]: I tried to improve this sentence (which was a mess even before Matthias tried to improve it, sorry).

**Comment by andreas-junghanns** [comment]
I fixed the merge conflicts caused by #1314.

**Comment by MBlesken** [comment]
@andreas-junghanns @chrbertsch this PR is ready for review now. Your as well as everybody else's comments are welcome. FYI @IZacharias

**Review by andreas-junghanns — APPROVED** [review]

**Comment by MBlesken** [comment]
As decided in #1308 added intermediateUpdatetime geq activationTime. FYI @chrbertsch @andreas-junghanns @IZacharias

**Review by IZacharias — APPROVED** [review]

**Pull request merged**